io_scene_nif/modules/nif_import/property/geometry/mesh.py

```diff
@@ -96,6 +97,7 @@ def process_property_list(self, n_block, b_mesh):
             processor.b_mesh = b_mesh
```

Comment:
If we are passing in the b_obj, we can avoid passing in the b_mesh and just get it using `b_mesh = b_obj.data`?

Reply:
Right, I just dropped that in as a hotfix since I needed the obj for the modifiers.
Down the line we can drop mat and mesh and get both from b_obj

Reply:
Mesh makes sense, not sure about mats as could have more than one?